rpc/client.go

```diff
@@ -119,6 +119,8 @@ type ClientInformational interface {
 	GetPeers(ctx context.Context) (InfoGetPeerResult, error)
 	// QueryBalance queries for balances under a given PurseIdentifier
 	QueryBalance(ctx context.Context, identifier PurseIdentifier) (QueryBalanceResult, error)
+	// QueryBalanceDetails query for full balance information using a purse identifier and a state identifier
+	QueryBalanceDetails(ctx context.Context, purseIdentifier PurseIdentifier, stateIdentifier BalanceStateIdentifier) (QueryBalanceDetailsResult, error)
```

Comment:
@ZhmakAS I see in other RPC methods that the pattern is to have multiple methods to query by block hash, height or latest. I would do the same here.
For `QueryBalance` it's also missing and has been reported in https://github.com/make-software/casper-go-sdk/issues/85.